Source/PPFSMain.c

```diff
@@ -7,6 +7,53 @@ void handler(int signum)
 
 int main(void)
 {
-	printf("IT WORKS!!!");
-	return 0;
-}
\ No newline at end of file
+	char *source=NULL;
+    char *destination=NULL;
+	int sleepTime = 5 * 60;
+	int recurency=0;
```

Comment:
"recursive" sounds better. Also, you could use <stdbool.h> and made it the actual Boolean, since we're using C99, which supports it by default.